frontend/micro-ui/web/micro-ui-internals/packages/modules/Estimate/src/pages/employee/EstimateInbox.js

```diff
@@ -2,11 +2,11 @@ import React from "react";
 import { useTranslation } from "react-i18next";
 import { Header, InboxSearchComposer } from "@egovernments/digit-ui-react-components";
 import inboxConfig from "../../configs/inboxConfig";
-
+import inboxConfigMukta from "../../configs/inboxConfigMukta";
 const EstimateInbox = () => {
     const { t } = useTranslation();
 
-    const configs = inboxConfig();
+    const configs = inboxConfigMukta(t);
```

Comment:
Integrate with mdms config

Reply:
This PR is for enhancements only . Will give a separate pr for inbox